x/ibc/04-channel/alias.go

```diff
@@ -2,103 +2,148 @@ package channel
-	NewMsgChannelOpenConfirm     = types.NewMsgChannelOpenConfirm
-	NewMsgChannelCloseInit       = types.NewMsgChannelCloseInit
-	NewMsgChannelCloseConfirm    = types.NewMsgChannelCloseConfirm
-	NewMsgPacket                 = types.NewMsgPacket
-	NewMsgTimeout                = types.NewMsgTimeout
-	NewMsgAcknowledgement        = types.NewMsgAcknowledgement
-	NewPacket                    = types.NewPacket
-	NewPacketAckCommitment       = types.NewPacketAckCommitment
-	NewPacketSequence            = types.NewPacketSequence
-	NewChannelResponse           = types.NewChannelResponse
-	DefaultGenesisState          = types.DefaultGenesisState
-	NewGenesisState              = types.NewGenesisState
-
-	// variable aliases
-	SubModuleCdc                 = types.SubModuleCdc
 	EventTypeChannelOpenInit     = types.EventTypeChannelOpenInit
 	EventTypeChannelOpenTry      = types.EventTypeChannelOpenTry
 	EventTypeChannelOpenAck      = types.EventTypeChannelOpenAck
 	EventTypeChannelOpenConfirm  = types.EventTypeChannelOpenConfirm
 	EventTypeChannelCloseInit    = types.EventTypeChannelCloseInit
 	EventTypeChannelCloseConfirm = types.EventTypeChannelCloseConfirm
 	AttributeValueCategory       = types.AttributeValueCategory
+	ErrInvalidLengthTypes        = types.ErrInvalidLengthTypes
+	ErrIntOverflowTypes          = types.ErrIntOverflowTypes
+	ErrUnexpectedEndOfGroupTypes = types.ErrUnexpectedEndOfGroupTypes
 )
 
-// nolint: golint
 type (
-	Keeper                 = keeper.Keeper
-	Channel                = types.Channel
-	Counterparty           = types.Counterparty
-	State                  = types.State
-	Order                  = types.Order
-	IdentifiedChannel      = types.IdentifiedChannel
-	ClientKeeper           = types.ClientKeeper
-	ConnectionKeeper       = types.ConnectionKeeper
-	PortKeeper             = types.PortKeeper
-	MsgChannelOpenInit     = types.MsgChannelOpenInit
-	MsgChannelOpenTry      = types.MsgChannelOpenTry
-	MsgChannelOpenAck      = types.MsgChannelOpenAck
-	MsgChannelOpenConfirm  = types.MsgChannelOpenConfirm
-	MsgChannelCloseInit    = types.MsgChannelCloseInit
-	MsgChannelCloseConfirm = types.MsgChannelCloseConfirm
-	MsgPacket              = types.MsgPacket
-	MsgAcknowledgement     = types.MsgAcknowledgement
-	MsgTimeout             = types.MsgTimeout
-	Packet                 = types.Packet
-	ChannelResponse        = types.ChannelResponse
-	PacketAckCommitment    = types.PacketAckCommitment
-	PacketSequence         = types.PacketSequence
-	GenesisState           = types.GenesisState
+	IdentifiedChannel             = types.IdentifiedChannel
+	ClientKeeper                  = types.ClientKeeper
+	ConnectionKeeper              = types.ConnectionKeeper
+	PortKeeper                    = types.PortKeeper
+	PacketAckCommitment           = types.PacketAckCommitment
+	PacketSequence                = types.PacketSequence
+	GenesisState                  = types.GenesisState
+	ChannelResponse               = types.ChannelResponse
```

Comment:
hmm I am not sure if this lint change is helpful at all since a `ChannelResponse` is much more specific than a `Response` within the channel package which houses a lot of types that could have a response.

Reply:
yeah, just add `// nolint`

Reply:
It's complaining because if this pkg is imported then it would be Channel.ChannelResponse. This is just duplicating the name. I would vote for just response in the alias